recipe/meta.yaml

```
@@ -55,29 +56,53 @@ requirements:
     - giflib {{ giflib }}
     - openjpeg {{ openjpeg }}
     - blosc 1.21.3
-    - lcms2 2.12
-    - libaec 1.0.4
+    - lcms2 2.16
+    - libaec 1
     - brotli 1.0.9
     - libzopfli 1.0.3
     - charls 2.2.0            # [not s390x]
     - snappy 1.1.9
```

```suggestion
    - snappy 1.2.1
```
https://github.com/cgohlke/imagecodecs/tree/v2024.9.22?tab=readme-ov-file#requirements

---

Done